main.go

```diff
@@ -655,13 +655,17 @@ func insertNewItems(w http.ResponseWriter, r *http.Request) {
 }
 
 func main() {
-	password := flag.String("password", "", "password for mysql db")
+	env := flag.String("env", "development", "Environment [development|staging|production]")
```

Comment:
let's keep the password flag, although i think i sort of see where you're going with this flag (which we can also keep if you give it more reason for being